Algorithm.Framework/QuantConnect.Algorithm.Framework.csproj

```diff
@@ -105,66 +105,36 @@
     <Compile Include="..\Common\Properties\SharedAssemblyInfo.cs">
       <Link>Properties\SharedAssemblyInfo.cs</Link>
     </Compile>
-    <Compile Include="Alphas\AlphaModel.cs" />
-    <Compile Include="Alphas\AlphaModelExtensions.cs" />
-    <Compile Include="Alphas\AlphaModelPythonWrapper.cs" />
     <Compile Include="Alphas\PearsonCorrelationPairsTradingAlphaModel.cs" />
-    <Compile Include="Alphas\CompositeAlphaModel.cs" />
     <Compile Include="Alphas\EmaCrossAlphaModel.cs" />
-    <Compile Include="Alphas\INamedModel.cs" />
-    <Compile Include="Alphas\NullAlphaModel.cs" />
     <Compile Include="Alphas\BasePairsTradingAlphaModel.cs" />
     <Compile Include="Alphas\RsiAlphaModel.cs" />
-    <Compile Include="Execution\ExecutionModel.cs" />
-    <Compile Include="Execution\IExecutionModel.cs" />
-    <Compile Include="Execution\ImmediateExecutionModel.cs" />
-    <Compile Include="Execution\NullExecutionModel.cs" />
-    <Compile Include="Execution\ExecutionModelPythonWrapper.cs" />
     <Compile Include="Execution\StandardDeviationExecutionModel.cs" />
     <Compile Include="Execution\VolumeWeightedAveragePriceExecutionModel.cs" />
-    <Compile Include="INotifiedSecurityChanges.cs" />
     <Compile Include="NotifiedSecurityChanges.cs" />
     <Compile Include="Portfolio\BlackLittermanOptimizationPortfolioConstructionModel.cs" />
-    <Compile Include="Portfolio\IPortfolioConstructionModel.cs" />
-    <Compile Include="Portfolio\IPortfolioOptimizer.cs" />
     <Compile Include="Portfolio\UnconstrainedMeanVariancePortfolioOptimizer.cs" />
     <Compile Include="Portfolio\MaximumSharpeRatioPortfolioOptimizer.cs" />
     <Compile Include="Portfolio\MeanVarianceOptimizationPortfolioConstructionModel.cs" />
     <Compile Include="Portfolio\MinimumVariancePortfolioOptimizer.cs" />
-    <Compile Include="Portfolio\NullPortfolioConstructionModel.cs" />
-    <Compile Include="Portfolio\PortfolioConstructionModel.cs" />
-    <Compile Include="Portfolio\PortfolioConstructionModelPythonWrapper.cs" />
     <Compile Include="Portfolio\EqualWeightingPortfolioConstructionModel.cs" />
     <Compile Include="Portfolio\ReturnsSymbolData.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="QCAlgorithmFramework.Python.cs" />
-    <Compile Include="QCAlgorithmFramework.cs" />
-    <Compile Include="QCAlgorithmFrameworkBridge.cs" />
     <Compile Include="Risk\CompositeRiskManagementModel.cs" />
-    <Compile Include="Risk\IRiskManagementModel.cs" />
     <Compile Include="Risk\MaximumDrawdownPercentPortfolio.cs" />
```

Comment:
I wonder if there's still added value in keeping a separate framework/algorithm project. I suppose w/ this change the framework project is more of a 'framework models' project.

Reply:
Exactly, in a previous commit, I actually deleted it entirely, but @jaredbroad pointed out we might keep it as a growing  'framework models' implementation project.